data_structures/binary_tree/binary_search_tree.py

```diff
@@ -152,26 +154,26 @@ def inorder(self, arr: list, node: Node):
     def find_kth_smallest(self, k: int, node: Node) -> int:
         """Return the kth smallest element in a binary search tree """
         arr = []
-        self.inorder(arr, node)  # append all values to list using inorder traversal
+        # append all values to list using inorder traversal
+        self.inorder(arr, node)
         return arr[k - 1]
 
-    def total_possible_BST(self, node:Node) -> int:
-        from math import factorial
+    def number_of_possible_binary_search_trees(self, node: Node) -> int:
         arr = []
         self.inorder(arr, node)
         n = len(arr)
         # Catalan number Cn = (2n)! / ((n + 1)! * n!)
         return factorial(2 * n) // factorial(n + 1) // factorial(n)
 
 
-
 def postorder(curr_node):
     """
     postOrder (left, right, self)
     """
     node_list = list()
     if curr_node is not None:
-        node_list = postorder(curr_node.left) + postorder(curr_node.right) + [curr_node]
+        node_list = postorder(curr_node.left) + \
```

Comment:
Please undo this useless change.  PEP8 cautions us not to use backslashes in Python code.

Reply:
Sure, I'm using black formatter, it did all these changes